addons/web/static/tests/views/search_view_tests.js

```diff
@@ -802,6 +802,78 @@ QUnit.module('Search View', {
         actionManager.destroy();
     });
 
+    QUnit.module('Search View Rendering');
+
+    QUnit.test('fields and filters with groups/invisible attribute are not always rendered but activable as search default', function (assert) {
+        assert.expect(15);
+
+        var actionManager = createActionManager({
+            actions: this.actions,
+            archs: this.archs,
+            data: this.data,
+            debug: true,
+        });
+
+        this.archs['partner,10,search'] =
+            "<search>" +
+                "<field name=\"display_name\" string=\"Foo B\" invisible=\"1\"/>" +
+                "<field name=\"foo\" string=\"Foo A\"/>" +
+                "<filter name=\"filterA\" string=\"FA\" domain=\"[]\"/>" +
+                "<filter name=\"filterB\" string=\"FB\" domain=\"[]\" invisible=\"1\" modifiers=\"{&quot;invisible&quot;: true}\"/>" +
+                "<filter name=\"filterC\" string=\"FC\" invisible=\"not context.get('show_filterC')\" modifiers=\"{&quot;invisible&quot;: false}\"/>" +
+                "<filter name=\"groupByA\" string=\"GA\" context=\"{'group_by': 'date_field:day'}\"/>" +
+                "<filter name=\"groupByB\" string=\"GB\" context=\"{'group_by': 'date_field:day'}\" invisible=\"1\" modifiers=\"{&quot;invisible&quot;: true}\"/>" +
+            "</search>"
+        this.data.partner.fields['display_name'] = {string: "Displayed name", type: 'char'};
+        this.actions[0].search_view_id = [10, 'search'];
+        this.actions[0].context = {
+            search_default_display_name: 'value',
+            search_default_filterB: true,
+            search_default_groupByB: true,
+            show_filterC: true,
+        };
+
+        actionManager.doAction(1);
+
+        // default filters/fields should be activated even if invisible
+        assert.strictEqual($('.o_searchview_facet').length, 3);
+
+        $('span.fa-filter').click();
```

Comment:
`$('span.fa-filter')` -> `actionManager.$('span.fa-filter')` (everywhere)

Reply:
actually, the control panel added no added as a child of the action manager (see screenshot below), so I used the controlPanel instance of the actionManager instead.

change being then:
`var controlPanel = actionManager.controlPanel;`
`$('span.fa-filter')` -> `controlPanel.$('span.fa-filter')`

![screenshot-999(527)](https://user-images.githubusercontent.com/2605529/84122948-7a4dd380-aa39-11ea-8311-e87b9a92fde0.png)

Reply:
Ho 12.0... ok sorry. You'll need to change that when it will be forwardported.